src/realm/sync/noinst/client_reset.cpp

```diff
@@ -631,22 +631,19 @@ void client_reset::recover_schema(const Transaction& group_src, Transaction& gro
     }
 }
 
-client_reset::LocalVersionIDs client_reset::perform_client_reset_diff(
-    const std::string& path_local, const util::Optional<std::array<char, 64>>& encryption_key,
-    sync::SaltedFileIdent client_file_ident, sync::SaltedVersion server_version, util::Logger& logger)
+client_reset::LocalVersionIDs client_reset::perform_client_reset_diff(DB& db, sync::SaltedFileIdent client_file_ident,
+                                                                      sync::SaltedVersion server_version,
+                                                                      util::Logger& logger)
 {
-    logger.info("Client reset, path_local = %1, "
-                "encryption = %2, client_file_ident.ident = %3, "
-                "client_file_ident.salt = %4, server_version.version = %5, "
-                "server_version.salt = %6 ",
-                path_local, (encryption_key ? "on" : "off"), client_file_ident.ident, client_file_ident.salt,
-                server_version.version, server_version.salt);
-
-    DBOptions shared_group_options(encryption_key ? encryption_key->data() : nullptr);
-    ClientHistoryImpl history_local{path_local};
-    DBRef sg_local = DB::create(history_local, shared_group_options);
-
-    auto group_local = sg_local->start_write();
+    //    logger.info("Client reset, path_local = %1, "
```

Comment:
Why'd this get commented out?

Reply:
I commented out all the logging to get things building and just missed this one when restoring them.